src/super_gradients/training/models/classification_models/base_classifer.py

```diff
@@ -47,19 +51,21 @@ def _get_pipeline(self, fuse_model: bool = True) -> ClassificationPipeline:
         )
         return pipeline
 
-    def predict(self, images: ImageSource, batch_size: int = 32, fuse_model: bool = True) -> ImagesClassificationPrediction:
+    def predict(self, images: ImageSource, batch_size: int = 32, fuse_model: bool = True, skip_image_resizing: bool = False) -> ImagesClassificationPrediction:
         """Predict an image or a list of images.
 
         :param images:      Images to predict.
         :param batch_size:  Maximum number of images to process at the same time.
         :param fuse_model:  If True, create a copy of the model, and fuse some of its layers to increase performance. This increases memory usage.
+        :param skip_image_resizing: If True, the image processor will not resize the images.
```

Comment:
If it's not supported in classification pipeline, should we maybe not expose it at all?

Reply:
The problem is that we because of the way we instantiate models (`model.get` which is dynamic, and task agnostic), we don't have a static contract for `model.predict`.
I don't know if I'm clear, but basically in practice we cannot know the parameters available for `model.predict` of a specific model in advance (unless we dig into the class inheritence...).

So I prefered to have an API that is as close as possible for every task, i.e. here also including `skip_image_resizing` for the classification model, and raise an error if passed.
It's a bit dirty I need to say, but at least the user would get a nice error message (`raise ValueError("`skip_image_resizing` is not supported for classification models.")`) instead of `parameter name not valid` (or something like that) which would be super confusing.
Open for discussion if you have a better idea